peachjam/templates/peachjam/taxonomy_detail.html

```diff
@@ -4,14 +4,38 @@
 {% block title %}{{ taxonomy.name }}{% endblock %}
 
 {% block page-title %}
-  <h1 class="mt-2">{{ taxonomy.name }}</h1>
+  <h1 class="my-4">{{ taxonomy.name }}</h1>
 {% endblock %}
 
+{% block breadcrumbs %}
+<nav style="--bs-breadcrumb-divider: url(&#34;data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='8' height='8'%3E%3Cpath d='M2.5 0L1 1.5 3.5 4 1 6.5 2.5 8l4-4-4-4z' fill='currentColor'/%3E%3C/svg%3E&#34;);"
```

Comment:
This shouldn't change the default breadcrumb style used elsewhere in the site.

Reply:
Also indentation is all crazy. :)

Reply:
done